.gitignore

```diff
@@ -11,4 +11,8 @@ Cargo.lock
 
 # Local testdrive environments, mainly for development
 testdrive/custom
-testdrive/assert
\ No newline at end of file
+testdrive/assert
+
+oura.toml
+
+oura-*.json
```

Comment:
I suggest we put `oura.toml` and `oura-*.json` inside `testdrive/custom` which is meant for local test files.

Reply:
ah ok sounds good